.changeset/twenty-ducks-pump.md

```diff
@@ -0,0 +1,6 @@
+---
+'@clerk/clerk-js': patch
+'@clerk/types': patch
+---
+
+Refactor `<Checkout />` components.
```

Comment:
this will need to be more descriptive so customers understand exactly what's being refactored here.